TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/RoutesToProfessionalStatus/AddRoute/_RouteDetail.cshtml

```diff
@@ -0,0 +1,115 @@
+@model RouteDetailViewModel
+<govuk-summary-list>
+    <govuk-summary-list-row>
+        <govuk-summary-list-row-key>Route</govuk-summary-list-row-key>
+        <govuk-summary-list-row-value>@Model.RouteToProfessionalStatus.Name</govuk-summary-list-row-value>
+    </govuk-summary-list-row>
+    <govuk-summary-list-row>
+        <govuk-summary-list-row-key>Status</govuk-summary-list-row-key>
+        <govuk-summary-list-row-value>@Model.Status.GetTitle()</govuk-summary-list-row-value>
+    </govuk-summary-list-row>
+
+    <govuk-summary-list-row>
+        <govuk-summary-list-row-key>Start date</govuk-summary-list-row-key>
+        <govuk-summary-list-row-value>@Model.TrainingStartDate?.ToString(UiDefaults.DateOnlyDisplayFormat)</govuk-summary-list-row-value>
+        <govuk-summary-list-row-actions>
+            <govuk-summary-list-row-action data-testid="add-start-date-link" href="@LinkGenerator.RouteAddStartDate(Model.PersonId, Model.JourneyInstanceId, Model.FromCheckAnswers)" visually-hidden-text="start date">Change</govuk-summary-list-row-action>
+        </govuk-summary-list-row-actions>
+    </govuk-summary-list-row>
+
+    <govuk-summary-list-row>
+        <govuk-summary-list-row-key>End date</govuk-summary-list-row-key>
+        <govuk-summary-list-row-value>@Model.TrainingEndDate?.ToString(UiDefaults.DateOnlyDisplayFormat)</govuk-summary-list-row-value>
+        <govuk-summary-list-row-actions>
+            <govuk-summary-list-row-action data-testid="add-end-date-link" href="@LinkGenerator.RouteAddEndDate(Model.PersonId, Model.JourneyInstanceId, Model.FromCheckAnswers)" visually-hidden-text="end date">Change</govuk-summary-list-row-action>
+        </govuk-summary-list-row-actions>
+    </govuk-summary-list-row>
+
+    @if (Model.AwardDateRequired != FieldRequirement.NotApplicable)
+    {
+        <govuk-summary-list-row>
+            <govuk-summary-list-row-key>Award date</govuk-summary-list-row-key>
+            <govuk-summary-list-row-value>@Model.AwardedDate?.ToString(UiDefaults.DateOnlyDisplayFormat)</govuk-summary-list-row-value>
+            <govuk-summary-list-row-actions>
+                <govuk-summary-list-row-action data-testid="add-award-date-link" href="@LinkGenerator.RouteAddAwardDate(Model.PersonId, Model.JourneyInstanceId, Model.FromCheckAnswers)" visually-hidden-text="award date">Change</govuk-summary-list-row-action>
+            </govuk-summary-list-row-actions>
+        </govuk-summary-list-row>
+    }
```

Comment:
Do we need `use-empty-fallback` on any of these rows?

Reply:
probably - done